xbmc/network/Network.h

```diff
@@ -145,10 +170,65 @@ class CNetwork
    void StartServices();
    void StopServices(bool bWait);
 
+   // tests if string is VALID IPv6 or VALID IPv4 address
```

Comment:
would be great if you could change these comments to doxygen style so they show up in the docs.